GSoC25/NEF/Bench.py

```
@@ -0,0 +1,293 @@
+    return {
+        "macro_precision": macro_p,
+        "macro_recall": macro_r,
+        "macro_f1": macro_f,
+        "avg_latency_ms": avg_latency_ms,  
+    }
+
+
+import re, os, json
+
+
+nef = EnhancedNEFPipeline(verbose=True)  
```

Ruff (0.14.2)

Undefined name `EnhancedNEFPipeline`

(F821)